tests/unit/Codeception/Util/MockAutoload.php

```diff
@@ -11,7 +11,7 @@ public static function setFiles(array $files)
         self::$files = $files;
     }
 
-    protected static function requireFile($file)
+    protected static function requireFile($file): bool
```

Comment:
string?

Reply:
I added this to fix the CI works.

Eventually, I'll continue with the `tests/` folder. I'll save all that fun for later.